src/dotnet/Chat.UI.Blazor/Components/Settings/AppSettings.razor

```diff
@@ -1,10 +1,35 @@
+@using ActualChat.Kvas
 @using ActualChat.Hosting
+@inherits ComputedStateComponent<AppSettings.Model>
+@{
+    var m = State.Value;
+    var status = m.IsAnalyticsEnabled
+        ? "Gathering statistics"
+        : "Turned off";
+}
+
 @if (NativeAppSettings != null) {
     <NativeAppSettingsView NativeAppSettings="@NativeAppSettings"></NativeAppSettingsView>
 }
 @if (HostInfo.HostKind != HostKind.MauiApp) {
     <SettingsBlazorModeToggle/>
 }
+<TileTopic Topic="How we use your data"/>
```

Comment:
I don't have a strong preference here. However I would probably name it something like: Data Usage Consent
And the button would be something like: Enhance Actual Chat with Data
With the description like: This setting allows us to use and process your information to enhance our services.